app/src/main/java/com/oracle/javafx/scenebuilder/app/SceneBuilderApp.java

```diff
@@ -668,7 +668,7 @@ private void performOpenFiles(List<File> fxmlFiles,
                     hostWindow.loadFromFile(fxmlFile);
                     hostWindow.openWindow();
                 }
-            } catch (IOException xx) {
+            } catch (RuntimeException | IOException xx) {
```

Comment:
Is there a reason not to use `Exception` superclass here, i.e. why `Runtime` and `IO` only?

Reply:
Very good point, yes, you are absolutely right. This is possibly a leftover from searching what was happening. Catching any kind of exception is a better solution here.

Reply:
If you wish to update this, happy to wait. Otherwise, it looks good to go.